src/styles/resources.scss

```diff
@@ -0,0 +1,11 @@
+.resources {
+  &__card-label {
+    color: $mustard-yellow;
+    margin-left: 5px;
+    margin-bottom: 25px;
+  }
+
+  &__rounded-card {
```

Comment:
does the cursor do the clicky hand thing when hovering? if not add cursor: pointer here

Reply:
no it doesn't -- added